crates/compute/src/tower.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024-2025 Irreducible Inc.
+
+//! Traits for working with field towers.
```

Comment:
I think we should probably just move `tower.rs` from `binius_core` to `binius_field` now. @GraDKh agree, disagree?

Reply:
Totally agree with that

Reply:
Removed and using field tower now.